pkg/auth/azure.go

```diff
@@ -140,7 +141,7 @@ func StreamInterceptor(
 
 func HttpAuthMiddleWare(resp http.ResponseWriter, req *http.Request, jwks *keyfunc.JWKS, clientId string, tenantId string) error {
 	token := req.Header.Get("authorization")
-	if req.URL.Path != "/" {
+	if strings.HasPrefix(req.URL.Path, "/environments") {
```

Comment:
please extract this into a constant and give it a name or comment
```
// all paths mentioned here DO NOT require authentication:
const ignoredAuthoPrefixes = ["..."]
```

Reply:
updated and moved the ignored paths to server.go so that paths are known there instead of here